crates/solidity/outputs/cargo/tests/src/doc_examples/simple_contract.rs

```diff
@@ -13,22 +13,22 @@ fn simple_contract() -> Result<()> {
     let parse_output = language.parse(ProductionKind::ContractDefinition, "contract Foo {}");
 
     let parse_tree = parse_output.tree();
+    let rule = parse_tree
```

Comment:
nit: `.as_rule_with_kind()` here, as you mentioned in another comment 😉

Reply:
I want to take ownership below (as unparse accepts an "owned" `Rc`) and using assert explicitly on the root rule kind, I feel, is more consistent with all the other asserts below.